### PR TITLE
add notes for how a project leaves the brigade

### PR DESCRIPTION
**Pull request by josephholsten**

### CODE REVIEW

File: departure.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Leaving Chef Brigade
+
+If a maintainer want to stop or leave, it's cool. We'll gladly point to the project's new home and keep an archive of the old history.
```
Comment (dblessing) [inline departure.md]: Minor edit: change 'want' to plural

File: departure.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## When the cookbook is for end-of-life tech
+
+If a cookbook is for software that's no longer feasable to keep running, like old open source, old commercial stuff, anything that can't be downloaded; then we update the cookbook `README.md` to mark as deprecated.
```
Comment (dblessing) [inline departure.md]: Feasible

**Comment by dblessing** [comment]
:+1:

File: departure.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+
+## When the cookbook is unused
+
+If a cookbook seems entirely unused, we may want to stop claiming it is maintained. In that case, we'll update the cookbook `README.md` to mark it as archived.
+
+## When the maintainer wants to leave
+
+If someone objects to the project moving and wants to continue maintainance under the brigade, we add a prominent link to fork on the readme and continue development. People are generally not idiots, so if they have a reason to leave us we should support them.
```
Comment (juliandunn) [inline departure.md]: typo on "maintenance"

**Comment by zuazo** [comment]
More than enough :+1:

**Comment by josephholsten** [comment]
@dblessing, @juliandunn: one more review pls

**Comment by juliandunn** [comment]
:shipit:

**Comment by dblessing** [comment]
:+1:

**Pull request merged**